hraven-core/src/main/java/com/twitter/hraven/rest/ObjectMapperProvider.java

```diff
@@ -108,13 +117,155 @@ public void serialize(Configuration conf, JsonGenerator jsonGenerator,
   }
 
   /**
+   * Custom serializer for TaskDetails object.
+   */
+  public static class TaskDetailsSerializer extends JsonSerializer<TaskDetails> {
```

Comment:
Any reason this has to be public, or can it become private? The same for JobDetailsSerializer.